### PR TITLE
Added ratelimit to login with username/password

### PR DESCRIPTION
**Pull request by tsteinruecken**

### CODE REVIEW

File: core/modules/user.py [hunk 1]
```diff
@@ -181,6 +183,9 @@ def login(self, name=None, password=None, skey="", *args, **kwargs):
 		if not name or not password or not securitykey.validate(skey, useSessionKey=True):
 			return self.userModule.render.login(self.loginSkel())
 
+		if not self.loginRateLimit.isQuotaAvailable():
+			raise errors.Forbidden()
```
Comment (sveneberth) [inline core/modules/user.py:187]: Can we raise a _429 Too Many Requests_ here and add the `Retry-After` header? This status code is more common and specified for this use case.

cf. [RFC 6585 Additional HTTP Status Codes](https://datatracker.ietf.org/doc/html/rfc6585#section-4), [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429).

**Review by sveneberth — CHANGES_REQUESTED** [review]

File: core/modules/user.py (hunk 1 above)
Comment (tsteinruecken) [reply core/modules/user.py:187]: That's consistent with other usages of the ratelimit module in ViUR (eg. https://github.com/viur-framework/viur-core/blob/main/core/modules/user.py#L244). But feel free to change this in a separate PR.

Comment (sveneberth) [reply core/modules/user.py:187]: Okay, i didn't now that we use the RateLimit already inside the core. So far this PR LGTM.

**Review by sveneberth — APPROVED** [review]

File: core/modules/user.py (hunk 1 above)
Comment (sveneberth) [reply core/modules/user.py:187]: Done with #463

**Pull request merged**